README.md

```diff
@@ -69,8 +69,24 @@ python manage.py runserver
 
 The "Before You Claim" page should load at [localhost:8000/before-you-claim/](http://localhost:8000/before-you-claim).
 
+### Fonts
+This application uses a proprietary licensed font (Avenir Next) that is not included in this repository.
+
+If you want to pull this font from a content delivery network (CDN), you can set
+[`@use-font-cdn`](https://github.com/cfpb/retirement/blob/master/src/css/main.less#L29)
+to `true` and rebuild the assets with `gulp build`.
+
+If you instead want to install self-hosted fonts locally, you can place the font files
+in `retirement_api/static/retirement/webfonts/` and restart the local web server.
+
+For Bureau employees or others with access to our private fonts repository,
+you can perform this step by creating a symbolic link to the font files:
+
+```sh
+ln -s ~/Projects/cfgov-fonts/fonts/ retirement_api/static/retirement/webfonts
```

Comment:
Shouldn't the cfgov-refresh font URL be `static.in/cfgov-fonts/fonts/` https://github.com/cfpb/cfgov-refresh/blob/3057cdfc7701193f993836f6a696f9ca027e11bd/docs/installation.md#webfonts ?

Reply:
That's what the installation directions say to do though, so shouldn't it follow it here?

Reply:
I've changed this wording to be more generic in 9cb88db, so as to support both cases: when you have the fonts when working with cfgov-refresh and when you have them somewhere else and are working on this repository standalone.